src/background/index.js

```diff
@@ -65,6 +65,18 @@ import {registerEvents, handleWidgetRemoved} from 'commerce/telemetry/extension'
     ['blocking', 'responseHeaders'],
   );
 
+  // Workaround for bug 1493470: Resend product info to the background
+  // script in case subframe loads clear the toolbar icon.
+  // TODO(osmose): Remove once Firefox 64 hits the release channel.
+  browser.webRequest.onCompleted.addListener(
+    (details) => {
+      if (details.tabId) {
+        browser.tabs.sendMessage(details.tabId, {type: 'resend-product'});
+      }
+    },
+    {urls: ['https://*/*', 'http://*/*'], types: ['sub_frame']},
+  );
```

Comment:
I recall you had originally had something like this before implementing the `resend` stuff -- why did you switch to `resend` originally?

Reply:
The issue is intermittent, and I suspect QA verified on a version that actually didn't have the fix (because I was slow to release it), then replicated the issue and re-filed. I'm not entirely sure how I was able to replicate it off of master, but at this point the fix is so consistent that I'm trying not to question it.